open-budget-il/main.py

```diff
@@ -677,22 +680,23 @@ def get_query(self):
         index = search.Index(name="OpenBudget")
         results = []
         try:
-            searchQueryStringList = [queryString]
```

Comment:
OK, I see what you mean.
So what about this (still simpler?) query:

מדינה AND ((type=BL and year=2014) OR (type=EN))

I'm trying to find a solution which for the general case (in which we don't have any query params) we fallback to a very simple query (i.e. just 'מדינה')